whylog_vim/output_formater/utils.py

```diff
@@ -0,0 +1,2 @@
+def convert_to_buttons(elem_list):
+    return '\n'.join(['[%s]' % seq for seq in elem_list])
```

Comment:
remove `[` and `]`. Only use them if you want to allocate a list. Here you just want to iterate.